setup.py

```diff
@@ -301,7 +301,7 @@ def run_tests(self):
             "doc8",
             "pyenchant",
             "readme_renderer >= 16.0",
-            "sphinx != 1.6.1",
+            "sphinx != 1.6.1, != 1.6.2",
```

Comment:
Work is ongoing. I don't really want to pin `< 1.6.1` because then we're trapped with that, and we drag anything else that wants to run our docs in the same env backwards once this is fixed.